### DISCUSSION TITLE
Date not showing when collapsed & Screen based settings

### DISCUSSION THREAD

**Discussion by max-grossenbacher** [dwarvesf/hidden#209] [Q&A]
Hi all, I have two questions:

1. I have been using hidden bar for a while, and only recently noticed, that when it's collapsed, the day and month are not shown. I cannot find a setting to change that. Is there a way?

2. Would it be possible that Hidden bar knows which screen I am using and only works when I use the smaller screen? For example, I have a MB Pro with notch, when I am using the notebook screen, hidden bar is on. When I plug in an external screen, without notch, hidden bar is not shown on this screen, only on the MacBook's notched screen?

**Comment by Coen83** [comment]
I have the same problem. Just installed hidden again but it sadly does not change anything

**Comment by Coen83** [comment]
@maximillion82, I found the answer to this problem.

Go to: System Preferences > Dock And Menu Bar > Clock > Show Date: set to "always"

Now it's visible again.

**Reply by muhozhrun** [reply]
thanks a lot

**Reply by priyanujkashyap** [reply]
Thanks, it worked.

**Reply by oooscars** [reply]
Thanks a bunch...
